internals/detect_intent_test.py

```diff
@@ -162,6 +162,17 @@ def test_detect_feature_id__alternative(self):
         5144822362931200,
         detect_intent.detect_feature_id(body))
 
+  def test_detect_feature_id__alternative(self):
```

Comment:
`test_detect_feature_id__alternative` already exists

Reply:
D'oh.   Editing error on my part.  That test method is an exact copy of the one above it.  The new code is below.